orca-web/src/main/groovy/com/netflix/spinnaker/orca/web/config/WebConfiguration.groovy

```diff
@@ -54,6 +56,11 @@ class WebConfiguration {
     return  new MetricsInterceptor(registry, "controller.invocations", ["application"], ["BasicErrorController"])
   }
 
+  @Bean
+  ManualJudgmentAuthzGroupsUtil mnlJudgUtil(Optional<Front50Service> front50Service) {
```

Comment:
Please do not truncate `mnl` and `Judg` when naming beans ... `s/mnlJudgUtl/ManualJugementAuthorizationUtil` (or something.

Reply:
done